### ISSUE TITLE
Hexagon Map | Keyboard | US Totals should be reachable using keyboard navigation only

### ISSUE THREAD

**Issue by LaurieSReynolds** [COVID19Tracking/website#1688]
Using the keyboard navigation, the US Totals hexagon element is not reachable.

steps
1) Tab to the map area
2) Use Arrow keys to navigate around the states

expected results
The US Totals hexagon should be reachable using keyboard navigation

actual results
The US Totals hexagon is not reachable

![ustotals-unreachable-tab](https://user-images.githubusercontent.com/18370991/102684506-0af63080-418e-11eb-8cca-4bd4e7f31730.gif)
